src/Razor/Microsoft.NET.Sdk.Razor/src/build/netstandard2.0/Microsoft.NET.Sdk.Razor.StaticWebAssets.targets

```diff
@@ -513,7 +513,7 @@ Copyright (c) .NET Foundation. All rights reserved.
        -->
       <_ExternalPublishStaticWebAsset
         Include="%(StaticWebAsset.FullPath)"
-        Condition="'%(SourceType)' != ''">
+        Condition="'%(SourceType)' != '' AND '%(StaticWebAsset.CopyToPublishDirectory)' != 'Never'">
```

Comment:
We don't want assets such as the boot.json produced during build to be copied to the publish directory. There was a bug in these targets where it wouldn't honor the CopyToPublishDirectory setting.

Reply:
We shouldn't leave random stuff in the SWA definition. What problem are we trying to solve here?